manual/Tasks/jar.html

```diff
@@ -217,6 +217,23 @@ <h3>Parameters</h3>
       include <samp>META-INF</samp> unless explicitly asked to. <em>Since Ant 1.8.0</em></td>
     <td>No; defaults to <q>false</q></td>
   </tr>
+  <tr>
+    <td>version</td>
+    <td>The <a href="https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/module/ModuleDescriptor.Version.html">module version</a>
+      to embed into the jar file's module descriptor, if it has one.  Not used in Java versions
+      older than Java 9.
```

Comment:
>Not used in Java versions older than Java 9.

I think this wording is confusing. Rephrasing it to "Not used in Java versions below Java 9", might be better.